lzero/model/unizero_world_models/world_model.py

```diff
@@ -850,7 +993,7 @@ def forward_recurrent_inference(self, state_action_history, simulation_index=0,
             self.latent_state,
             is_init_infer=False,
             simulation_index=simulation_index,
-            latent_state_index_in_search_path=latent_state_index_in_search_path
+            search_depth=[]
```

Comment:
why set empty here

Reply:
search_depth is not used in this method, polished now.